packages/comet-extras/src/components/toggle/toggle.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './toggle.style.css';
+import { ChangeEvent, ChangeEventHandler, useEffect, useState } from 'react';
+
+export interface ToggleProps {
+  /**
+   * The unique identifier for this component
+   */
+  id: string;
+  /**
+   * The name of the text input
+   */
+  name?: string;
+  /**
+   * Whether the toggle is checked or not
+   */
+  checked?: boolean;
+  /**
+   * A label to display with the toggle
+   */
+  label?: string;
+  /**
+   * An accessible label for the toggle
+   */
+  ariaLabel?: string;
+  /**
+   * Custom callback for when input is changed
+   */
+  onChange?: ChangeEventHandler<HTMLInputElement>;
+}
+
+export const Toggle = ({ id, name, checked = false, label, ariaLabel, onChange }: ToggleProps) => {
+  const [isChecked, setIsChecked] = useState(false);
+  const toggleHandler = (event: ChangeEvent<HTMLInputElement>): void => {
+    setIsChecked(!isChecked);
+    if (onChange) {
+      onChange(event);
+    }
+  };
+
+  useEffect(() => {
+    setIsChecked(checked);
+  }, [checked]);
+
+  return (
+    <div className="toggle">
+      <label htmlFor={id} className="toggle-label" tabIndex={0} aria-label={ariaLabel}>
+        <div className="relative">
```

Comment:
This class should be replaced with a toggle specific class, the updated to include `position: relative`

Reply:
Added **.toggle .pos-rel { position: relative;}** class